ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -38,17 +43,36 @@ function transformEnumResponse(metacardTypes, response) {
         case 'SHORT': //needed until enum response correctly returns numbers as numbers
           result[key] = value.map((
             subval //handle cases of unnecessary number padding -> 22.0000
-          ) => Number(subval))
+          ) => {
+            let numObj = {}
+            numObj['label'] = Number(subval)
+            numObj['value'] = Number(subval)
+            return numObj
+          })
+          break
+        case 'COUNTRY':
+          result[key] = value.map(subval => {
+            let countryObj = {}
+            countryObj['label'] = `${subval} - ${countries[key][subval]}`
+            countryObj['value'] = subval
+            return countryObj
+          })
           break
         default:
-          result[key] = value
+          result[key] = value.map(subval => {
+            let defaultObj = {}
+            defaultObj['label'] = subval
+            defaultObj['value'] = subval
+            return defaultObj
+          })
           break
       }
       return result
     },
     {}
   )
 }
+let countries = {}
```

Comment:
❓ What's the purpose of declaring `countries` outside of the exported module? Seems it could be done inside, no?